Edited the release notes of the Mantid Workbench changes for the 6.16.0 release of Mantid (#41604)
mantidproject/mantid: docs/source/release/v6.16.0/mantidworkbench.rst

Bugfixes
--------
- (`#41303 <https://github.com/mantidproject/mantid/pull/41303>`_) The :ref:`Error Bars Settings <PlotSettings>` (CapSize, Cap Thickness, Width, Error Every) are now correctly applied when plotting a spectrum with errors.
- (`#41295 <https://github.com/mantidproject/mantid/pull/41295>`_) The :ref:`Colorfill Plots <Colorfill_Plots>` now support over plotting with a ``TableWorkspace``, issuing a warning instead of raising an exception.
- (`#41515 <https://github.com/mantidproject/mantid/pull/41515>`_) The :ref:`Algorithm toolbox <WorkbenchAlgorithmToolbox>` now allows populating algorithms without the previously open dialog menus crashing.
- (`#41389 <https://github.com/mantidproject/mantid/pull/41389>`_) The :ref:`Messages Window <WorkbenchMessagesWindow>` now prints logs that are more readable in both *light* and *dark* mode at all logging levels.

> - (`#41083 <https://github.com/mantidproject/mantid/pull/41083>`_) The :ref:`Superplot <WorkbenchSuperplot>` documentation is launched when the plot is ``Superplot`` and the *Help* button is clicked.

Reviewer: ```suggestion
- (`#41295 <https://github.com/mantidproject/mantid/pull/41295>`_) :ref:`Colorfill Plots <Colorfill_Plots>` now support over plotting with a ``TableWorkspace``, issuing a warning instead of raising an exception.
- (`#41515 <https://github.com/mantidproject/mantid/pull/41515>`_) The :ref:`Algorithm toolbox <WorkbenchAlgorithmToolbox>` now allows populating algorithms without the previously open dialog menus crashing.
- (`#41389 <https://github.com/mantidproject/mantid/pull/41389>`_) The :ref:`Messages Window <WorkbenchMessagesWindow>` now prints logs that are more readable in both *light* and *dark* mode at all logging levels.
- (`#41083 <https://github.com/mantidproject/mantid/pull/41083>`_) The :ref:`Superplot <WorkbenchSuperplot>` documentation is now correctly launched when the plot is a ``Superplot`` and the *Help* button is clicked.
```

InstrumentViewer
----------------

New features
############

> - (`#41169 <https://github.com/mantidproject/mantid/pull/41169>`_) In the **new** :ref:`Instrument Viewer <InstrumentViewer>`, peak selection no longer requires constantly clicking buttons for adding and deleting peaks. Now peak selection is triggered with a peak selection mode that lets you continually select detectors and add (left mouse click) and delete (right mouse click) peaks without extra clicking or moving the mouse around.

Reviewer: I feel all these new instrument view updates could be in a sub-list like

```
- New features added to the **new** Instrument Viewer:
 
  - (#link one) change one
  - (#link two) change two
```

Also, we shouldn't be linking to the `<InstrumentViewer>` reference since this goes to the docs about the old instrument viewer I believe. I don't think the new one has a doc page yet.

Author: One of the release notes from a PR was linking to it, so that is why I updated it to be consistent across the other bullet points and that is why I ended up emphasising the word **new** because I suspected that the documentation might not be the updated one.

I feel that because we have the heading of Instrument Viewer and sub heading of New Features, the sentence might be redundant but let me have a look at this part of the release notes again.

- (`#41263 <https://github.com/mantidproject/mantid/pull/41263>`_) Plots can now be normalised by :math:`Q^{-4}` (provided the X dimension is in :math:`Q`) via the right-click menu.
- (`#40996 <https://github.com/mantidproject/mantid/pull/40996>`_) The Help documentation now opens in the system's default web browser instead of an embedded viewer, improving ``Qt6`` compatibility. The ``qtwebengine`` dependency has been removed. This change was necessary in order to resolve issues with the render limitations of ``qttextbrowser`` and to ensure that the help documentation is displayed correctly across different platforms.

Bugfixes
--------
- (`#41303 <https://github.com/mantidproject/mantid/pull/41303>`_) The :ref:`Error Bars Settings <PlotSettings>` (CapSize, Cap Thickness, Width, Error Every) are now correctly applied when plotting a spectrum with errors.
- (`#41295 <https://github.com/mantidproject/mantid/pull/41295>`_) :ref:`Colorfill Plots <Colorfill_Plots>` now support over plotting with a ``TableWorkspace``, issuing a warning instead of raising an exception.

> - (`#41515 <https://github.com/mantidproject/mantid/pull/41515>`_) The :ref:`Algorithm toolbox <WorkbenchAlgorithmToolbox>` now allows populating algorithms without the previously open dialog menus crashing.

Reviewer: Sorry I didn't spot this at first but this edit for Colorfill plots over plotting has lost the original meaning. Previously, if you dragged a table workspace onto the plot there would be a crash (or at least an error); now it's just a warning. I wouldn't say 'Colorfill Plots now support over plotting with a TableWorkspace', I think that gives the wrong impression.

Also, I don't think the next release note (41515) is specific to the Algorithm toolbox since the crash was occurring in the Start Live Data widget.

Author: I replaced the word **support** with **allow** and rephrased the next release note as well.

- (`#41389 <https://github.com/mantidproject/mantid/pull/41389>`_) The :ref:`Messages Window <WorkbenchMessagesWindow>` now prints logs that are more readable in both *light* and *dark* mode at all logging levels.
- (`#41083 <https://github.com/mantidproject/mantid/pull/41083>`_) The :ref:`Superplot <WorkbenchSuperplot>` documentation is now correctly launched when the plot is a ``Superplot`` and the *Help* button is clicked.

InstrumentViewer
----------------

New features
############
- Updates to the **new** Instrument Viewer:
    - (`#41169 <https://github.com/mantidproject/mantid/pull/41169>`_) Peak selection no longer requires constantly clicking buttons for adding and deleting peaks. Now peak selection is triggered with a peak selection mode that lets you continually select detectors and add (left mouse click) and delete (right mouse click) peaks without extra clicking or moving the mouse around.
    - (`#41437 <https://github.com/mantidproject/mantid/pull/41437>`_) The Detector Info box now shows up at the bottom and is hidden when no detectors are selected or when too many detectors (more than 3) are selected. This should reduce the visual clutter on the GUI.

>     - (`#41184 <https://github.com/mantidproject/mantid/pull/41184>`_) The "Draw Shapes" option is now stored in the Mantid properties file, and the option is applied when the view is opened. This means that if you check "Draw Shapes", close the view, and open it again, it will still be checked and shapes will be drawn. The option is stored under the key "InstrumentView.DrawShapes".

Reviewer: ```suggestion
    - (`#41184 <https://github.com/mantidproject/mantid/pull/41184>`_) The ``Draw Shapes`` option is now stored in the Mantid properties file, and the option is applied when the view is opened. This means that if you check ``Draw Shapes``, close the view, and open it again, it will still be checked and shapes will be drawn. The option is stored under the key ``InstrumentView.DrawShapes``.
```

    - (`#40841 <https://github.com/mantidproject/mantid/pull/40841>`_) A new ellipse widget for selecting detectors has been added.
    - (`#40850 <https://github.com/mantidproject/mantid/pull/40850>`_) An option to draw detectors using their actual geometric shapes instead of as points has been added.
    - (`#41001 <https://github.com/mantidproject/mantid/pull/41001>`_) An option to display counts on a log scale is added.
    - (`#40830 <https://github.com/mantidproject/mantid/pull/40830>`_) New reset buttons for the contour and integration range sliders have been added.
    - (`#40811 <https://github.com/mantidproject/mantid/pull/40811>`_) It displays the component tree from the IDF and allows you to select one or more components in that tree.
    - (`#41221 <https://github.com/mantidproject/mantid/pull/41221>`_) It now updates the counts data when the integration range has min == max.

>     - (`#41054 <https://github.com/mantidproject/mantid/pull/41054>`_) It now allows Groupings and Masks to be saved as `.cal` files.

Reviewer: ```suggestion
    - (`#40811 <https://github.com/mantidproject/mantid/pull/40811>`_) Now displays the component tree from the IDF and allows you to select one or more components in that tree.
    - (`#41221 <https://github.com/mantidproject/mantid/pull/41221>`_) Counts data is now updated when the integration range ``min == max``.
    - (`#41054 <https://github.com/mantidproject/mantid/pull/41054>`_) Now allows Groupings and Masks to be saved as ``.cal`` files.
```

    - (`#41348 <https://github.com/mantidproject/mantid/pull/41348>`_) The projection is flipped along the beam axis (from the instrument) instead of the z-axis.

> - (`#41315 <https://github.com/mantidproject/mantid/pull/41315>`_) The :ref:`ISIS Reflectometry Interface <interface-isis-refl>` now supports the **new** Instrument Viewer as an option.

Reviewer: ```suggestion
- (`#41315 <https://github.com/mantidproject/mantid/pull/41315>`_) The :ref:`ISIS Reflectometry Interface <interface-isis-refl>` now supports the **new** Instrument Viewer as an option available in the settings.
```

Bugfixes
############
- Issues resolved for the **new** Instrument Viewer:
    - (`#40834 <https://github.com/mantidproject/mantid/pull/40834>`_) The mouse wheel events on the projection and units combo boxes are no longer triggered when the mouse is hovering over them. This prevents the projection and units from being changed unintentionally when scrolling with the mouse wheel.
    - (`#41051 <https://github.com/mantidproject/mantid/pull/41051>`_) It no longer crashes when a ``LOQ`` workspace is open and side-by-side projection is selected.
    - (`#41221 <https://github.com/mantidproject/mantid/pull/41221>`_) It no longer crashes when the integration range is edited through text boxes and window is closed.
    - (`#41413 <https://github.com/mantidproject/mantid/pull/41413>`_) It no longer crashes when masking all detectors.

>     - (`#41201 <https://github.com/mantidproject/mantid/pull/41201>`_) It exits without an error message when the interface is closed while certain operations are still processing.

Reviewer: ```suggestion
    - (`#41051 <https://github.com/mantidproject/mantid/pull/41051>`_) Now no longer crashes when a ``LOQ`` workspace is open and side-by-side projection is selected.
    - (`#41221 <https://github.com/mantidproject/mantid/pull/41221>`_) Now no longer crashes when the integration range is edited through text boxes and window is closed.
    - (`#41413 <https://github.com/mantidproject/mantid/pull/41413>`_) now no longer crashes when masking all detectors.
    - (`#41201 <https://github.com/mantidproject/mantid/pull/41201>`_) Now exits without an error message when the interface is closed while certain operations are still processing.
```

Author: I personally prefer **It** over **Now** but I suppose it is a stylistic choice.

Reviewer: happy for a gatekeeper to weigh in and choose one

Author: It is okay. I have already pushed the changes. I think this is a subjective task. I made less changes to my release notes because they are phrased the way I think. But that does not mean that other peoples way of writing is not equally valid.
